### ISSUE TITLE
Previous sourcemaps are lost if file has a trailing newline

### ISSUE THREAD

**Issue by jessepollak** [hughsk/uglifyify#20]
The [regular expression used to pull out the sourcemaps](https://github.com/hughsk/uglifyify/blob/master/index.js#L42) from previous files fails if the file has a trailing new line.

This causes the sourcemaps to be lost and developers to be sad.

Happy to submit a PR with a fix — let me know. I've currently worked around it by adding

```coffee
bundler.transform global: true, -> (es.map (file, cb) -> cb null, file.toString().trim())
```

**Comment by hughsk** [comment]
Should be fixed as of `3.0.1`!

**Comment by jessepollak** [comment]
Terrific, thank you :)
